### PR TITLE
Exercise: Sequence Interface

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Closed [#334](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/334)

### CODE REVIEW

File: 01-syntax/05-oop/Sequence.js [hunk 1]
```diff
@@ -1 +1,52 @@
-// → Your code here.
+function Interface(arr) {
```
Comment (nesterone) [inline 01-syntax/05-oop/Sequence.js]: Too much code  for that kind on task 

I want only 'Sequence' in that file, also use 

This one

https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/05-oop/Sequence.js#L1

**Comment by artem-konovalov-personal** [comment]
I have changed code structure. Now it doesn't contain Interface constructor and his prototype as in the previous version of the code

File: 01-syntax/05-oop/Sequence.js [hunk 2]
```diff
@@ -1 +1,39 @@
...
+  return undefined;
+};
+
+function RangeSeq(start, end) {
+  var obj = new Sequence([]);
+  obj.fillObjArray(start, end);
+  return obj;
+}
+
+function logFive(obj) {
```
Comment (nesterone) [inline 01-syntax/05-oop/Sequence.js]: Any ideas on how to reuse `toString` for such kind of task ?

**Comment by artem-konovalov-personal** [comment]
I have change the code structure, added `toString` method for implementing the iteration

File: 01-syntax/05-oop/Sequence.js [hunk 3]
```diff
@@ -1,37 +1,42 @@
...
-    this.arr.push(start);
-    arguments[0]++;
+  var newStart = start;
+
+  while (newStart <= end) {
+    this.arr.push(newStart);
+    newStart += 1;
   }
+
+  return undefined;
```
Comment (nesterone) [inline 01-syntax/05-oop/Sequence.js]: Is it really necessary to return `undefined`, What the reason ?

**Comment by artem-konovalov-personal** [comment]
I have just removed unnecessary `return` statement

**Pull request merged**